assets/javascripts/audit_log.js

```diff
@@ -8,10 +8,20 @@ function loadAuditLogTable ()
 {
     $('#audit_log_table').DataTable({
         lengthMenu: [20, 40, 100],
-        processing: true,
         serverSide: true,
         search: {search: searchquery},
-        ajax: {url: ajax_url, type: "GET", dataType: 'json'},
+        ajax: {
+            url: ajax_url,
+            type: 'GET',
+            dataType: 'json',
+            error: function(xhr) {
+                var msg = 'Search resulted in error ' + xhr.status;
```

Comment:
If the web UI is completely down this results in "Search resulted in error 0" which is not so nice. Also "Search" is a little bit over-specific because filtering and using the pagination will run into the same error handler. Maybe "Table query …" would be better.